item/historical/scripts/T001.ipynb

```diff
@@ -562,7 +562,7 @@
      "text": [
       ">> Number of countries missing values: 16\n",
       ">> Countries missing values:\n",
-      "['Greece', 'Hungary', 'Slovak Republic', 'Switzerland', 'Serbia, Republic of', 'Czech Republic', 'Liechtenstein', 'North Macedonia', 'Belarus', 'Austria', 'Montenegro, Republic of', 'Moldova', 'Armenia', 'Slovenia', 'Luxembourg', 'Lithuania']\n",
+      "['Liechtenstein', 'North Macedonia', 'Slovenia', 'Czech Republic', 'Slovak Republic', 'Lithuania', 'Luxembourg', 'Austria', 'Moldova', 'Hungary', 'Belarus', 'Armenia', 'Switzerland', 'Serbia, Republic of', 'Greece', 'Montenegro, Republic of']\n",
```

Comment:
Ditto comment on T000. Is this a meaningful change, or just an unsorted list that has changed order?